README.md

```diff
@@ -9,11 +9,11 @@ For more information, see the [Wiki](https://github.com/microsoft/DirectXShaderC
 Visit the [DirectX Landing Page](https://devblogs.microsoft.com/directx/landing-page/) for more resources for DirectX developers.
 
 ## Downloads
-You can download the latest successful build's artifacts (built by Appveyor) for the master branch:
+You can download the latest successful build's artifacts (built by Appveyor) for the nain branch:
```

Comment:
Just a comment but I think this should be "main"

Reply:
Thanks Grace! :)